modules/README.md

```diff
@@ -44,14 +44,15 @@ that will be passed to the Puppet run.
 You can test the module without running a pxp-agent. Make sure the module is
 executable and you can run it like:
 
+### Posix
 ```
 $ sudo echo "{\"params\":{\"env\":[],\"flags\":[\"--noop\"]}, \"config\" : {\"puppet_bin\" : \"/opt/puppetlabs/bin/puppet\"}}" | puppet run
 ```
 
-### NOTE
-
-This manner of execution is subject to change on completion of
-[CTH-299](https://tickets.puppetlabs.com/browse/CTH-299)
+### Windows
```

Comment:
You might want to note this is for `cmd.exe`, since there are multiple interpreters with different escaping rules on Windows (i.e. PowerShell and Cygwin)